docs/SETUP.md

```diff
@@ -458,3 +458,98 @@ You can check the logs with this command
 ```
 journalctl -u ilp-kit -f -n 500
 ```
+## Run with Forever
+This is alternative way of systemd setup. 
+
+Install "forever" with the command, if you don't have "forever" package yet. 
+``` 
+sudo npm install forever -g
+```
+
+Create a site.sh file 
+```
+nano site.sh
+```
+
+with the contents
+```
+#!/bin/bash
+
+start() {
+    echo -n "Starting forever: "
+	forever start  --workingDir=/var/ilpkit  -c "npm start" -a --uid ledger /var/ilpkit
```

Comment:
`/var/ilpkit` should be changed to an environment variable (or a variable in the script) to make it easier to configure